woocommerce/payment-gateway/class-sv-wc-payment-gateway-payment-form.php

```diff
@@ -977,7 +977,11 @@ public function render_payment_fields() {
 	 */
 	protected function render_payment_field( $field ) {
 
-		woocommerce_form_field( $field['name'], $field, $field['value'] );
+		woocommerce_form_field(
+			isset ( $field['name'] ) ? $field['name'] : null,
+			$field,
+			isset ( $field['value'] ) ? $field['value'] : null
```

Comment:
Any particular reason you added a space after `asset()`?

```suggestion
			isset( $field['name'] ) ? $field['name'] : null,
			$field,
			isset( $field['value'] ) ? $field['value'] : null
```

Reply:
😑 no particular reason... just lack of attention. Thanks for pointing that! Fixed in https://github.com/skyverge/wc-plugin-framework/pull/515/commits/ea2dcbbff107e1f854da76a57d5438d5555b4602.